### DISCUSSION TITLE
not working as of 1.8.2

### DISCUSSION THREAD

**Discussion by DesertFade** [JustOptimize/return-ShowHiddenChannels#45] [Q&A] [answered]
Hello,

i have followed the instructions to install the plugin.
Currently i'm using v 1.8.2 of BetterDiscord.
ZeresPluginLibrary and this plugin from this repository is installed.
Though all of this, i don't see any hidden channels.

Is there any known issue or is it me doing something wrong? I've reinstalled it and the library a few times, re-activated and so on but nothing works out.

**Comment by JustOptimize** [comment]
The only known problem is that you cannot see hidden channels in the server you were on when you started Discord.
What version of discord are you using? And on what OS?
Do you see any errors in the console?

**Comment by JustOptimize** [comment]
Do you have the `SplitLargeMessages` plugin? If so, try disabling it and tell me if it solves the problem

**Comment by DesertFade** [comment, marked as answer]
Okay, not it seems to be working. Not sure what exactly fixed it, i turned my PC on this morning and it works.
I'm on version stable 153481 (c00173e), Windows 10 64 bit.
I'm not sure if you mean the F12 console but i did not get any errors so far shown to me.

I had SplitLargeMessages installed, i think it was removed after the big betterdiscord update. I reinstalled it for testing purposes, seems to be working also after a restart (so no issue).
